tests/foreman/endtoend/test_api_endtoend.py

```diff
@@ -896,6 +906,11 @@ class AvailableURLsTestCase(TestCase):
     def setUp(self):
         """Define commonly-used variables."""
         self.path = f'{settings.server.get_url()}/api/v2'
+        if is_open('BZ:1887932'):
+            missing = '/katello/api/activation_keys/:activation_key_id/subscriptions'
+            API_PATHS['subscriptions'] = tuple(
+                path for path in API_PATHS['subscriptions'] if path != missing
+            )
```

Comment:
could make this open for any future additions
```suggestion
        missing = []
        if is_open('BZ:1887932'):
            missing.append('/katello/api/activation_keys/:activation_key_id/subscriptions')
        API_PATHS['subscriptions'] = tuple(
            path for path in API_PATHS['subscriptions'] if path not in missing
        )
```

Reply:
Going to do it with a dict though, so its not just subscriptions that can be added to.